deployments/mainnet/weth/migrations/1666906000_configurate_seed_reserves_and_rewards.ts

```diff
@@ -0,0 +1,132 @@
+import { DeploymentManager, migration } from '../../../../plugins/deployment_manager';
+import { calldata, exp, getConfigurationStruct, proposal } from '../../../../src/deploy';
+
+import { expect } from 'chai';
+
+const cETHAddress = '0x4ddc2d193948926d02f9b1fe9e1daa0718270ed5';
+const COMPAddress = '0xc00e94cb662c3520282e6f5717214004a7f26888';
+
+export default migration('1666906000_configurate_seed_reserves_and_rewards', {
+  prepare: async (deploymentManager: DeploymentManager) => {
+    return {};
+  },
+
+  enact: async (deploymentManager: DeploymentManager) => {
+    const trace = deploymentManager.tracer();
+    const ethers = deploymentManager.hre.ethers;
+
+    const comptrollerV2 = await deploymentManager.fromDep('comptrollerV2', 'mainnet', 'usdc');
+    const cometFactory = await deploymentManager.fromDep('cometFactory', 'mainnet', 'usdc');
+    const {
+      governor,
+      comet,
+      configurator,
+      cometAdmin,
+      rewards,
+      WETH,
+      wstETH,
+      cbETH,
+    } = await deploymentManager.getContracts();
+
+    const configuration = await getConfigurationStruct(deploymentManager);
+
+    const actions = [
+      // 1. Set v2 cETH speeds to 0
+      {
+        contract: comptrollerV2,
+        signature: '_setCompSpeeds(address[],uint256[],uint256[])',
+        args: [[cETHAddress], [0], [0]],
+      },
+
+      // 2. Set the factory in the Configurator
+      {
+        contract: configurator,
+        signature: 'setFactory(address,address)',
+        args: [comet.address, cometFactory.address],
+      },
+
+      // 3. Set the configuration in the Configurator
+      {
+        contract: configurator,
+        signature: 'setConfiguration(address,(address,address,address,address,address,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint104,uint104,uint104,(address,address,uint8,uint64,uint64,uint64,uint128)[]))',
```

Comment:
@jflatow I think you have 1 extra parenthesis at the end, since `Configuration` should be `(..., AssetConfig[])`, not `(..., AssetConfig[]))`